//Let not declared

try {
  console.log(a) ;
  let a = 3
}
catch {
  console.log ("Let must be declared before use")
};
